src/pages/vagas/index.tsx

```diff
@@ -1,4 +1,4 @@
-import { Footer, Header, JobsCard } from '../../components';
+import { Footer, Header, JobsCard, JobsSearchForm } from '../../components';
 import Image from 'next/image';
 import { GoTo } from '../../components';
```

Comment:
So notei agora, mas se puder remover essa linha de importação para junto da que ja esta importando os componentes

```suggestion
import { Footer, Header, JobsCard, JobsSearchForm, GoTo } from '../../components';
import Image from 'next/image';
```